index.md

```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
 
 ### Automated Tests
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+Automated tests automatically check code does what the developer intends. Learning to write automated tests will free the domain experts from having to constantly re-test changes by hand.
+
+**Writing automated tests will eliminate new bugs and speed up collaboration.**
+
+Automated tests are one of the most powerful tools in a developers toolkit. My motto forged from years of practical experience is _code that isn't tested does not work_ as expected
+
+This seminar will explain all the ways in which automated tests are essential. It will cover how to write tests, even when the code is complicated, with plenty of examples.
+
+#### Seminar Syllabus
+
+The seminar can cover:
+
+ * What tests are.
+ * Why tests are valuable.
+ * How to write tests (including when it is tricky).
+ * When to write tests.
+ * Running tests
+
+Software & Technologies:
+
+ * [Python](https://www.python.org/){:target="_blank"} - a popular programming language
+ * [Python unittest](https://docs.python.org/3/library/unittest.html){:target="_blank"} - the built-in unit testing framework in Python
+ * [Python unittest.mock](https://docs.python.org/3/library/unittest.mock.html){:target="_blank"} - the built-in mocking framework in Python
+
+This seminar can be tweaked to use the language of your choice, or cover testing examples that resemble the kind of code you write to make this easier to apply. [Get in touch](mailto:hi@tkiley.co.uk) today to start writing automated tests today.
 
 ### "Good" Code
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+Writing code is relatively easy, but writing "good" code is
+elusive. By following industry approaches over time the quality of the code will improve.
```

Comment:
"over time the quality of the code will improve" feels a bit of a rubbish goal - maybe pick something more flashy for your opener!

Reply:
Think about making this a bit punchier - e.g. improve the code you write tomorrow, which will in turn improve your whole code base